mlflow/utils/rest_utils.py

```diff
@@ -31,6 +31,9 @@
         504,  # Gateway Timeout
     ]
 )
+_MAX_RETRIES = 5
+_BACKOFF_FACTOR = 2
+_DEFAULT_TIMEOUT = 120
```

Comment:
How do we customize these values?

Reply:
That would be great. Is there any documentation that needs to be added for the environment variables?

Reply:
According to @harupy 's suggestion,
We can create a module like this and put all the environment variable names in it:
"""
This module defines environment variable names used in MLflow.
"""

#: foo
MLFLOW_FOO = "MLFLOW_FOO"

#: bar
MLFLOW_BAR = "MLFLOW_BAR"

...
and let sphinx generate a document for this module.

Reply:
@peterdhansen @WeichenXu123 Very excited to merge this once we introduce environment variables. Please let me know if there's anything I can do to help.

Reply:
+1 on using environment variables to override